lerobot-web-fe/src/models/teleoperate.model.ts

```
@@ -0,0 +1,18 @@
+export interface TeleoperateResponse {
+  status: string;
+  message: string;
+  additionalProp1: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)